spec/datetimeformat.html

```diff
@@ -754,11 +754,11 @@ <h1>Intl.DateTimeFormat.prototype.resolvedOptions ()</h1>
       </emu-table>
 
       <p>
-        For web compatibility reasons, if the property hourCycle is set, the hour12 property should be set to *true* when hourCycle is *"h11"* or *"h12"*, or to *false* when hourCycle is *"h23"* or *"h24"*.
+        For web compatibility reasons, if the property *"hourCycle"* is set, the *"hour12"* property should be set to *true* when *"hourCycle"* is *"h11"* or *"h12"*, or to *false* when *"hourCycle"* is *"h23"* or *"h24"*.
       </p>
 
       <emu-note>
-        In this version of the ECMAScript 2020 Internationalization API, the timeZone property will be the name of the default time zone if no timeZone property was provided in the options object provided to the Intl.DateTimeFormat constructor. The first edition left the timeZone property *undefined* in this case.
+        In this version of the ECMAScript 2020 Internationalization API, the *"timeZone"* property will be the name of the default time zone if no *"timeZone"* property was provided in the options object provided to the Intl.DateTimeFormat constructor. The first edition left the *"timeZone"* property *undefined* in this case.
```

Comment:
I'm not certain how far you want to go in this PR, but I suppose <code>\`Intl.DateTimeFormat\` constructor</code> might be ideal in the end.

Reply:
I think it would have to be a different PR -- because I intentionally didn't change it, actually. Many (if not all...?) global objects in ECMA-262 are not styled, like `Array` here for example: https://tc39.es/ecma262/#sec-array-objects

Maybe we should change it in both locations.

Reply:
Oh wow, you're quite right, it really is quite inconsistent.
I'll ask about this on IRC and maybe create a 262 issue for it!

Reply:
Just for visibility: https://github.com/tc39/ecma262/pull/1860

Reply:
Cool! If this goes through on 262 I'll make a PR on 402.